tensorflow_probability/python/sts/dynamic_regression_test.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2018 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Dynamic Linear Regression State Space Model Tests."""
```

Comment:
Note that that harness checks that the default parameter priors have batch shape matching the observed time series, to ensure we're exposing somewhat sane semantics by default. For `LinearRegression` we had to hack around this since it doesn't take an observed time series argument, but since this component does, I think things should `just work`. So you can likely just copy the simpler formulation used by other components, e.g.,
https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/sts/structural_time_series_test.py#L252

Reply:
Cool - I missed this so thanks for helping. The test was useful too as it pointed out a mistake I had made. Done in 99218d7